src/features/profile/components/show-skills.tsx

```diff
@@ -0,0 +1,35 @@
+import Typography from "@mui/material/Typography";
+import Box from "@mui/material/Box";
+import { Skill } from "core/profiles/types";
+
+const ShowSkills = () => {
+  const skills: Skill[] = [
+    {
+      id: 1,
+      name: "Desarrollador Front-end",
+    },
+    {
+      id: 2,
+      name: "Desarrollador Back-end",
+    },
+    {
+      id: 3,
+      name: "React",
+    },
+    {
+      id: 4,
+      name: "TypeScript",
+    },
+  ];
+  return (
+    <Box>
+      {skills.map((item: Skill) => (
+        <Typography key={item.id} variant="h6">
+          {item.name}
+        </Typography>
```

Comment:
Encapsular todo el componente hijo
```suggestion
        <SkillComponent key={item.id} skill={item} />
```

Reply:
Si, lo que pasa es que diria que eso va un poco de la mano con los estilos (?) por eso no lo hice asi, pero si esta contemplado

Reply:
lo dejamos asi por ahora? y el siguiente sprint cuando tome en cuenta los estilos lo cambio? o lo cambio de una vez?

Reply:
Dale!, dejalo para el siguiente sprint (este) xD

Reply:
XD